packages/web-console/src/scenes/Editor/Shortcuts/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React from "react"
+import styled from "styled-components"
+import { color } from "../../../utils"
+import { Text } from "../../../components"
+import { platform } from "../../../utils"
+
+type ShortcutsList = { keys: string[][]; title: string }[]
```

Comment:
how come this is not clashing with https://github.com/questdb/ui/pull/28/files#diff-863cb95c15f7ab7369c679049c640042f4833c05f55ad42b09dd8ae13f474101R122

Reply:
TypeScript magic from what I remember 🪄

Reply:
perhaps lets avoid that magic?

cause reading this i need a sec to comprehend wtf https://github.com/questdb/ui/pull/28/files#diff-863cb95c15f7ab7369c679049c640042f4833c05f55ad42b09dd8ae13f474101R122-R128